src/pages/patientView/copyNumberAlterations/column/CohortColumnFormatter.tsx

```diff
@@ -72,4 +88,52 @@ export default class CohortColumnFormatter
             );
         });
     }
+
+    private static getGisticValue(data:DiscreteCopyNumberData, gisticData:IGisticData)
+    {
+        const gistic = gisticData[data.entrezGeneId];
+        let summary:IGisticSummary|undefined;
+
+        if (gistic)
+        {
+            // here we are assuming that we have at most 2 values in the GisticSummary array:
+            // one for amp === true, and one for amp === false
+            summary = gistic.find((gs:IGisticSummary) => {
+                // alteration === 2 => amplified
+                // otherwise => not amplified (deleted)
+                return gs.amp === (data.alteration === 2);
+            });
+        }
+
+        return summary === undefined ? null : summary;
+    }
+
+    private static makeGisticIcon(qValue:number, peakGeneCount:number)
+    {
+        return (
+            <DefaultTooltip
+                placement="right"
+                overlay={CohortColumnFormatter.getGisticTooltip(qValue, peakGeneCount)}
+                arrowContent={<div className="rc-tooltip-arrow-inner"/>}
+            >
+                <svg width="12" height="12">
+                    <circle r="5" cx="6" cy="6" stroke="#55C" fill="none"/>
+                    <text x="3" y="8.5" fontSize="7" fill="#66C">
+                        G
+                    </text>
+                </svg>
+            </DefaultTooltip>
+        );
+    }
+
+    private static getGisticTooltip(qValue:number, peakGeneCount:number)
+    {
+        return (
+            <div>
+                <span style={{fontWeight:'bold'}}>Gistic</span><br/>
```

Comment:
Just copied from mutation table's cohort renderer. Actually, it might be better to have a shared component for this "icon + tooltip" thing.